fe/pages/dashboards.py

```diff
@@ -11,6 +14,121 @@
     icon="bi-bar-chart-line",
 )
 
-layout = _iframe.layout(
-    "https://lookerstudio.google.com/embed/reporting/86ab32f9-151b-4f91-87eb-060a22f2f890/page/QJFZE"
+DEPMAP_DATA = pd.read_parquet(
+    os.path.abspath("./") + "/parquet_data_sources" + "/depmap_metadata.parquet")
```

Comment:
Suggestion: If you import `from pathlib import Path`, this can be rewritten in a modern Pythonic style like

```python
Path(".") / "parquet_data_sources" / "depmap_metadata.parquet"
```

(Same below)